corehq/apps/reports/standard/sms.py

```diff
@@ -113,9 +105,10 @@ def _fmt(val):
             )
         ]
 
-def _sms_count(user, startdate, enddate, message_type='SMSLog'):
+
+def _sms_count(user, startdate, enddate):
```

Comment:
Might be nice to sync `VerifiedNumber` to SQL too, and then `_sms_count` and `len(get_verified_numbers)` can be replaced with a single query ... but not in this PR I don't think.